spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -116,8 +116,12 @@ protected long waitUntilAfter(long timestampMillis) {
     return current;
   }
 
+  protected String sqlFormat(String query, Object... args) {
```

Comment:
Why is this needed? Formatting SQL is done by the `sql` method. Even if it is reused, that's the right place for it. Can you roll back this change?

Reply:
Reverted.